storage/metamorphic: TestPebbleEquivalence failed

storage/metamorphic.TestPebbleEquivalence [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18769111?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18769111?buildTab=artifacts#/) on master @ [bbaa2e50b2fe789527aac09b99fa5eee432e7695](https://github.com/cockroachdb/cockroach/commits/bbaa2e50b2fe789527aac09b99fa5eee432e7695):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestPebbleEquivalence (15m0s)
	TestPebbleEquivalence/engine/random=3 (12m24s)

```
Stack: 

```
goroutine 472014 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
output file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=54052287015/output.meta
store path = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=524288390/store
=== RUN   TestPebbleEquivalence/engine/standard=0
check file = 
output file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence3391437827/output.meta
store path = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=03307245929/store
=== RUN   TestPebbleEquivalence/engine/standard=13
check file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence3391437827/output.meta
output file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=13356367174/output.meta
store path = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=131592183699/store
=== RUN   TestPebbleEquivalence/engine/standard=15
check file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence3391437827/output.meta
output file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=152702720077/output.meta
store path = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=15632882944/store
=== RUN   TestPebbleEquivalence/engine/standard=14
check file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence3391437827/output.meta
output file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=143798895043/output.meta
store path = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=141337802101/store
=== RUN   TestPebbleEquivalence/engine/random=1
check file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence3391437827/output.meta
output file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_random=11970051719/output.meta
store path = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_random=12314433875/store
=== RUN   TestPebbleEquivalence/engine/standard=1
check file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence3391437827/output.meta
output file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=1777828573/output.meta
store path = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=12870269139/store
=== RUN   TestPebbleEquivalence/engine/standard=11
check file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence3391437827/output.meta
output file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=11523865537/output.meta
store path = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=112160317126/store
=== RUN   TestPebbleEquivalence/engine/standard=12
check file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence3391437827/output.meta
output file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=124022221951/output.meta
store path = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=123194319349/store
=== RUN   TestPebbleEquivalence/engine/standard=8
check file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence3391437827/output.meta
output file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=8742664988/output.meta
store path = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=82033445313/store
=== RUN   TestPebbleEquivalence/engine/standard=9
check file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence3391437827/output.meta
output file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=93556871927/output.meta
store path = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_standard=92939102452/store
=== RUN   TestPebbleEquivalence/engine/random=0
check file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence3391437827/output.meta
output file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_random=0923510635/output.meta
store path = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_random=04249275034/store
=== RUN   TestPebbleEquivalence/engine/random=3
check file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence3391437827/output.meta
output file = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_random=31273540046/output.meta
store path = /artifacts/tmp/_tmp/ed77015fcf5315e89955a74e01226740/TestPebbleEquivalence_engine_random=33560241445/store
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #139886 storage/metamorphic: TestPebbleEquivalence failed [A-storage C-test-failure O-robot T-storage branch-release-25.1]
</p>
</details>
/cc @cockroachdb/storage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestPebbleEquivalence.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-47354

--- a/pkg/storage/metamorphic/meta_test.go
+++ b/pkg/storage/metamorphic/meta_test.go
@@ -30,7 +30,7 @@ var (
 	check        = flag.String("check", "", "run operations in specified file and check output for equality")
 	inMem        = flag.Bool("in-mem", false, "use an in-memory filesystem")
 	compareFiles = flag.String("compare-files", "", "comma-separated list of output files to compare; used by TestCompareFiles")
-	opCount      = flag.Int("operations", 20000, "number of MVCC operations to generate and run")
+	opCount      = flag.Int("operations", 10000, "number of MVCC operations to generate and run")
 )
 
 type testRun struct {

